app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -1,121 +1,434 @@
+          we perform a first preprocessing step to convert the FASTQ into FASTA files.
+        </p>
+        <pre style="max-height: initial;">
+<!--   --><b>$</b> umgap fastq2fasta A1.fq A2.fq | tee preprocessed.fa
+<!--   -->>read1/1
+<!--   -->ATCGCGCACGCGGCCGATGCCCCAGAAGAGATTGACAGCGGTGGGGCGGGCGGCGGCGAGGTGGTCGCAGATCTCGGCGACCTCTGCGTTGAGGGTCGGG
+<!--   -->>read1/2
+<!--   -->AAGATGGCGACTGGATGATGATCGCGCGGCAGGCCACCATCTACGATCCCGCAGTGAAGCATTACTAACCATGATGCGCAACGACTCGCTGTCAGAGCTA
+<!--   -->>read2/1
+<!--   -->AGATTGCTGGTGCGGGTGCTCTGCCGGGCTTCTTCATCTCGGACCGCGCATCCGATGCGCACACGGCACAGTAGGTATGCGGTGAGAGCACCTCGCTTTT
+<!--   -->>read2/2
+<!--   -->CGCAATCTTGCGGCGCACCGCGATCAAATCGGGATAGTCCGGTGTGTAACGCGTGCTCAGGTCATCTGCCCGTACCCGCAATACATTCAACTGTGTTTTA
+<!--   -->>read3/1
+<!--   -->CCCTCAAGCCCGATCGAACTTTCCTACTGCCCGACCTCAGCGCGGAGCATTACCGAATCGTCGTCAACAATCTCCCCGATGGCTTCTATGTGAACTCCAT
+<!--   -->>read3/2
+<!--   -->CAAAATGGAGTGGGCGCTACTGCTCCGTGAGCAGGGTCAGTTGGAGGGATTGGGGCGTGCGCTCGGGGATGCTAATGGCGGTGCCCCTGGATTCCTGAGA
+        </pre>
+        <p>
+          This command will interleave the given FASTQ files and output a FASTA stream,
+          in which the paired-ends alternate each other.
+        </p>
+      </div>
+    </div>
 
-    if line.startswith('>'):
-        header = line.split('_')[0]
-        if header != prevheader:
-            prevheader = header
-            sys.stdout.write('{}\n'.format(header))
-        else:
-            skipped += 1
-            sys.stderr.write('{}: skipped {}\n'.format(skipped, header.rstrip()))
-    else:
-        sys.stdout.write(line)</pre>
-        <p>
-          <b>NOTE</b>: The file <code>singles_proteins.fasta.faa</code> may contain two mates coming from the same clone, but that are not consecutive in the file. Since we know that the protein fragments from both mates come from the same organism, the processing could be improved if we also join the peptides from the protein fragments of the same clone. This is not yet covered by the python script. This could be done by sorting the results after the next step (after <code>unipept pept2lca</code>).
-    </p>
-        <p>
-          Then we compute the LCA for all individual tryptic peptides <i>(timing: 37 mins 17 secs)</i>.
-    </p>
-        <pre>
-<b>$</b> unipept pept2lca < peptides.fst > peptides.lca.fst</pre>
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Gene prediction</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          To convert the metagenomics reads into a metaproteomics problem,
+          we have to perform some kind of gene prediction.
+          While any gene prediction tool can be used, UMGAP was tested with FragGeneScan (2010) and FragGeneScan++,
+          our in-house improved version of FragGeneScanPlus (2015), a predictor using on Hiddem Markov Models.
+          On the other hand you can choose not to bother with prediction
```

Comment:
`Alternatively, you can also try all possible translations and use the built-in six-frame translation tool.`

_[Original comment](https://github.ugent.be/unipept/unipept/pull/642#discussion_r16546) by @bmesuere on Mon Aug 20 2018 at 13:41._